index.html

```diff
@@ -2316,12 +2342,6 @@ <h3>Production</h3>
 to the value representation rules in the list above.
         </p>
 
```

Comment:
@selfissued @msporny would this be acceptable?
```suggestion
        </p>
        <p class="note" title="@context">
"Please note that the value of the @context object member will be ignored, if present."
        </p>

```